Loop/Managers/RemoteDataServicesManager.swift

```diff
@@ -644,18 +642,22 @@ extension RemoteDataServicesManager {
             UserDefaults.appGroup?.deleteQueryAnchor(for: remoteDataService, withRemoteDataType: .overrides)
         }
     }
+}
+
+extension RemoteDataServicesManager: RemoteCommandSource {
     
```

Comment:
This is async as someday the service will make an API request to put a remote lock on the command state (i.e. Set to in-progress state)